index.js

```diff
@@ -65,30 +66,37 @@ app.post('/post', (req, res) => {
 
 	//Create new post
 	db.collection('post').save(req.body, (err, result) => {
-	    // if (err) return console.log(err);
-
-	    // res.json(result);
 	    if(err) {
     		response = { error: true, message: "Error adding data" };
-  		} else {
+  		} 
+  		else {
     		response = { error: false, message: "Data added", id: result._id };
```

Comment:
maybe fix the spacing here
`{
        error: true,
        message: "Data added",
        id: result._id
}`

Reply:
Fixed!